JuiceMaker/JuiceMaker/Controller/ChangeStockViewController.swift

```diff
@@ -8,22 +8,80 @@
 import UIKit
 
 class ChangeStockViewController: UIViewController {
-
+    var vcon: ViewController? = nil
```

Comment:
p2;

`ViewController` 를 나타내기 위한 축약어 vcon을 사용하고 있고 프로퍼티의 이름만 봤을 때 어떤 역할을 할지 알기 어려울 것으로 보이네요 👀
Swift API Design Guidelines의 [Use Terminology Well](https://www.swift.org/documentation/api-design-guidelines/#use-terminology-well)에 따르면 축약어의 사용을 지양하는 것이 더 읽히기 쉽다고 되어 있어요!
축약하지 않은 형태 및 좀 더 해당 프로퍼티의 역할(모달을 띄운 화면)을 잘 나타낼 수 있는 네이밍을 사용해보면 어떨까요?

> Avoid abbreviations. Abbreviations, especially non-standard ones, are effectively terms-of-art, because understanding depends on correctly translating them into their non-abbreviated forms.
> Name variables, parameters, and associated types according to their roles, rather than their type constraints.